### PR TITLE
Hi-fi: 404 page

### LINKED ISSUE #11: Pull Data: 404 Page

### LINKED ISSUE #53: Hi-fi: 404 Page

### PR DESCRIPTION
**Pull request by isabellberzsenyi**
This PR contains the code for the 404 page.
**Please note adjustments will be made once the Navigation gets mered**

### CODE REVIEW

File: components/four-oh-four-page/FourOhFourPage.style.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+  @media ${max.tabletSm} {
+    text-align: left;
+  }
+`;
+
+const ErrorText = styled(P)`
+  margin-top: clamp(1em, 2vw, 2em);
+  width: 100%;
+  line-height: 1.6em;
```
Comment (jzhzhang66) [inline components/four-oh-four-page/FourOhFourPage.style.js]: I feel like we could take out this line and just use the default line height from P to keep text consistent across the site

**Review by jzhzhang66 — COMMENTED** [review]
Everything looks really good!! Just a few comments -

From screen size around 1450 - 1200, the image compresses. Its a minor issue so if you don't have time to fix it no worries 
image
image

File: pages/404.js [hunk 2]
```diff
@@ -1,30 +1,22 @@
...
+export default function FourOhFour({ navigationData, errorPageData }) {
   return (
-    <>
-      <H1>{getString(errorPageTitle)}</H1>
-      <P>{getString(errorText)}</P>
-      <Link href="/">
-        <P>Click here to redirect to home page!</P>
-      </Link>
-    </>
+    <FourOhFourPage
```
Comment (jean-zhang) [inline pages/404.js:6]: Are we planning on adding the footer too?

**Review by jean-zhang — APPROVED** [review]

File: pages/404.js (hunk 2 above)
Comment (isabellberzsenyi) [reply pages/404.js:6]: from my chat with jay I don't think so

File: pages/404.js [hunk 3]
```diff
@@ -1,30 +1,22 @@
...
-
-export default function FourOhFour(errorPageData) {
-  const {
-    error_page_title: errorPageTitle,
-    error_text: errorText,
-  } = errorPageData.errorPageData;
+import { get404, getNavigation } from '../lib/api';
+import FourOhFourPage from '../components/four-oh-four-page/FourOhFourPage';
 
+export default function FourOhFour({ navigationData, errorPageData }) {
```
Comment (josephmannis) [inline pages/404.js:4]: lol

**Review by josephmannis — APPROVED** [review]
Looks good!

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
For some screen sizes I think the title is cut off b/c of the nav - 
image

For mobile even though its not cut off could we also move everything down a bit to make it more centered? 
image

**Review by jzhzhang66 — APPROVED** [review]
LOOKS GOOOD

**Pull request merged**